src/components/Hashcat/Hashcat.tsx

```diff
@@ -41,8 +18,22 @@ interface FormValuesType {
     maxPwdLen: number;
 }
 
-const inputType = ["Hash Value", "File"];
-const attackMode = ["Straight", "Brute-force", "Hybride Wordlist + Mask"];
+const title = "Hashcat";
+const description_userguide = `
+    Hashcat is an advanced password recovery tool that provides brute-force attacks that are conducted with the hash values of passwords that are either guessed or applied by the tool. DDT currently supports 3 attack modes including Straight, Brute-force, and Hybrid Wordlist + Mask.
+
+    Step-by-Step Guide:
+    1. Pick an attack mode.
+    2. Input hash type and hash algorithm code.
+    3. Input the hash value.
+    4. Input the password file.
+    5. Add additional commands as needed.
+    6. Click Scan to commence.
+    7. View the output below.
+`;
+
+const inputTypeOptions = ["Hash Value", "File"];
+const attackModeOptions = ["Straight", "Brute-force", "Hybrid Wordlist + Mask"];
 
```

Comment:
You should add useEffect

```suggestion
    // Check if the command is available and set the state variables accordingly.
    useEffect(() => {
        // Check if the command is available and set the state variables accordingly.
        checkAllCommandsAvailability(dependencies)
            .then((isAvailable) => {
                setIsCommandAvailable(isAvailable); // Set the command availability state
                setOpened(!isAvailable); // Set the modal state to opened if the command is not available
                setLoadingModal(false); // Set loading to false after the check is done
            })
            .catch((error) => {
                console.error("An error occurred:", error);
                setLoadingModal(false); // Also set loading to false in case of error
            });
    }, []);
```